JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -30,7 +30,7 @@
         }
     };
 
-    var method = ["Event", "Exception", "Metric", "PageView", "Trace"];
+    var method = ["Event", "Exception", "Metric", "PageView", "Trace", "Ajax"];
```

Comment:
should we notify Arun to update the version of JS snippet for Visual Studio 2015 Update 1?